### PR TITLE
add Contributions page to FAQ section

### PR DESCRIPTION
**Pull request by redecvi**
## Summary
This PR adds a Contributions section to the FAQ

## Details
The new section describes What contributions are, how they are earned, and how they are accumulated.

ticket:
https://app.zenhub.com/workspaces/obol-workspace-648c1170ee94656741353327/issues/gh/obolnetwork/data-pipelines/112

### CODE REVIEW

File: docs/faq/contributions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- **Lido / Mellow**: The Obol vault accepts ETH deposits into the Simple DVT Module, bridging robust institutional staking and decentralized native staking. [Learn more](https://lido.fi).
+- **Chorus One / Stakewise**: Stakewise and Chorus One offer a platform for ETH staking with integrated Obol DVs. [Learn more](https://opus.chorus.one/pool/stake).
+- **Stakely**: Our partner enabling contributions through DV staking. [Learn more](https://stakely.io/).
+
+## How Contributions are Accumulated
+
+Contributions are accumulated based on the staking rewards generated by your validators (pubkeys). Here's a quick overview of the process:
+
+1. **Staking Rewards**: Each DV contributes 1% of its (consensus layer) staking rewards to the Obol Collective. This means the more effective and uptime your validators have, the more rewards (and thus contributions) you will generate.
```
Comment (anadi2311) [inline docs/faq/contributions.md:28]: I think we should be more descriptive here on how DV contributes 1% using splits and how validator rewards gets converted to operator contributions

i.e when clusters deploy splits with 1 % to  retroactivefunding.obol.eth,  validator CL rewards are rolled up into clusters with  and then split using split percentages to operators. 

You can also include the formulas with summations etc to make it easier to understand.

We should also mention contributions are updated once a day using of this formula.

We should include that Launchpad only shows direct contributions by Squads and Clusters. Users who contributed through partners will have to go through their UI to find contributions

File: docs/faq/contributions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 6
+description: Earning Obol Contributions
+---
+
+# Obol Contributions Program
+
+## What are Contributions?
+
+The Obol Contributions Program acknowledges and rewards participants for their efforts in scaling and decentralizing the Ethereum network by staking on Distributed Validators (DVs). When you stake with DVs, 1% of the (consensus layer) staking rewards are directed to the Obol Collective’s “1% for Decentralization” retroactive funding program (RAF), and the portion of rewards you contribute to the RAF reflects your contributions.
```
Comment (OisinKyne) [inline docs/faq/contributions.md]:
```suggestion
The Obol Contributions Program acknowledges and rewards participants for their efforts in scaling and decentralising the Ethereum network by staking on Distributed Validators (DVs). When you stake with DVs, 1% of the staking rewards are directed to the Obol Collective’s “1% for Decentralisation” retroactive funding program (RAF), and the amount of rewards you contribute to the RAF program represents your contribution.
```

File: docs/faq/contributions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## How to Earn Contributions
+
+You can earn contributions by:
+
+1. **Running your own DV Cluster**: You can create and manage your own DV cluster using our Launchpad. This allows you to directly participate in the network's decentralization efforts and earn contributions. To get started, visit our [DV Launchpad](https://launchpad.obol.org). You can also visit our [Squad Staking](https://squadstaking.com) page for inspiration and to find cluster-mates.
```
Comment (OisinKyne) [inline docs/faq/contributions.md:16]:
```suggestion
1. **Running your own DV Cluster**: You can create and manage your own DV cluster using our Launchpad. This allows you to directly participate in the network's decentralisation efforts and earn contributions. To get started, visit our [DV Launchpad](https://launchpad.obol.org). You can also visit our [Squad Staking](https://squadstaking.com) page for inspiration and to find squad members.
```

File: docs/faq/contributions.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+2. **Staking through Staking Partners**: By participating in staking through our partners, your staking rewards will also contribute to the Obol Collective. Here’s how you can start earning contributions through our partners:
+
+- **Lido / Mellow**: The Obol vault accepts ETH deposits into the Simple DVT Module, bridging robust institutional staking and decentralized native staking. [Learn more](https://lido.fi).
```
Comment (OisinKyne) [inline docs/faq/contributions.md:20]: Probably should be a mellow URL or they won't get contributions if they just deposit through the lido front door.

File: docs/faq/contributions.md (hunk 1 above)
Comment (OisinKyne) [inline docs/faq/contributions.md]:
```suggestion
1. **Staking Rewards**: Each DV contributes 1% of its staking rewards to the Obol Collective. This means the higher effectiveness and uptime your validators have, the more rewards (and thus contributions) you will generate.
```

File: docs/faq/contributions.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- **Chorus One / Stakewise**: Stakewise and Chorus One offer a platform for ETH staking with integrated Obol DVs. [Learn more](https://opus.chorus.one/pool/stake).
+- **Stakely**: Our partner enabling contributions through DV staking. [Learn more](https://stakely.io/).
+
+## How Contributions are Accumulated
+
+Contributions are accumulated based on the staking rewards generated by your validators (pubkeys). Here's a quick overview of the process:
+
+1. **Staking Rewards**: Each DV contributes 1% of its (consensus layer) staking rewards to the Obol Collective. This means the more effective and uptime your validators have, the more rewards (and thus contributions) you will generate.
+2. **Tracking and Recognition**: Your contributions are tracked off-chain in a centralized database. You can query your contributions using our API endpoints or view them on the Obol DV Launchpad and our partners' staking platforms. [Learn more](https://api.obol.tech/docs)
```
Comment (OisinKyne) [inline docs/faq/contributions.md]:
```suggestion
2. **Tracking and Recognition**: Your contributions are tracked off-chain in a centralised database. You can query your contributions using our API endpoints or view them on the [Obol DV Launchpad](https://launchpad.obol.org/cluster/list/) and our partners' staking platforms. [Learn more](https://docs.obol.org/api)
```

**Review by OisinKyne — APPROVED** [review]
Small changes, but looks good to me!

File: docs/faq/contributions.md (hunk 4 above)
Comment (redecvi) [reply docs/faq/contributions.md]: Right now, there doesn't seem to be a prod mellow/lido URL available. Once we have it (before the launch), I'll add it

File: docs/faq/contributions.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- **Lido / Mellow**: The Obol vault accepts ETH deposits into the Simple DVT Module, bridging robust institutional staking and decentralised native staking. [Learn more](https://lido.fi).
+- **Chorus One / Stakewise**: Stakewise and Chorus One offer a platform for ETH staking with integrated Obol DVs. [Learn more](https://opus.chorus.one/pool/stake).
+- **Stakely**: Our partner enabling contributions through DV staking. [Learn more](https://stakely.io/).
+
+## How Contributions are Accumulated
+
+Your DV earns rewards which are distributed to operators based on the predefined percentages in the split contract. Each DV contributes 1% of its staking rewards to the Obol Collective. This means that the higher the effectiveness and uptime of your validators, the more rewards (and thus contributions) you will generate.
+
+## Tracking and Recognition
```
Comment (anadi2311) [inline docs/faq/contributions.md:28]: We should also add another section that describes
1. This is the first wave of staking partners
2. If someone wants to include their protocol in contribution program, how can they connect with us.

File: docs/faq/contributions.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+Your contributions are tracked off-chain in a centralised database. Contributions are updated daily using the aforementioned formula. You can query your contributions using our [API](https://docs.obol.org/api) or view them on the [Obol DV Launchpad](https://launchpad.obol.org/cluster/list/) and our partners' staking platforms. Note that the Launchpad only shows direct contributions by Squads and Clusters. Users who contributed through partners will need to access their partner’s UI to find their contributions.
```
Comment (anadi2311) [inline docs/faq/contributions.md:30]: replace https://docs.obol.org/api
by 
https://docs.obol.org/api#tag/Address/operation/AddressController_getAddressContributions

Comment (anadi2311) [reply docs/faq/contributions.md:30]: > Note that the Launchpad only shows direct contributions by Squads and Clusters

Note that the Launchpad and API only shows direct contributions by Squads and Clusters

**Review by anadi2311 — CHANGES_REQUESTED** [review]

File: docs/faq/contributions.md [hunk 8]
```diff
@@ -25,12 +25,6 @@ You can earn contributions by:
 
 Your DV earns rewards which are distributed to operators based on the predefined percentages in the split contract. Each DV contributes 1% of its staking rewards to the Obol Collective. This means that the higher the effectiveness and uptime of your validators, the more rewards (and thus contributions) you will generate.
```
Comment (anadi2311) [inline docs/faq/contributions.md:26]: Let's also include with formula contributions are denominated in Gwei

**Review by anadi2311 — CHANGES_REQUESTED** [review]

File: docs/faq/contributions.md (hunk 4 above)
Comment (slugmann321) [reply docs/faq/contributions.md]: likely it'll be https://app.mellow.finance/restake/ethereum-dvsteth, though not yet 100% finalized

**Review by anadi2311 — APPROVED** [review]

**Pull request merged**